tests/models/clip/test_modeling_clip.py

```diff
@@ -746,6 +955,115 @@ def test_model_from_pretrained(self):
         model = CLIPModel.from_pretrained(model_name)
         self.assertIsNotNone(model)
 
+    @parameterized.expand([("float16",), ("bfloat16",), ("float32",)])
+    @require_torch_sdpa
+    @slow
+    @is_flaky()
+    def test_eager_matches_sdpa_inference(self, torch_dtype: str):
+        super().test_eager_matches_sdpa_inference(
+            torch_dtype=torch_dtype,
+            logit_keys=("logits_per_image", "logits_per_text"),
+            use_attention_mask_options=(None, "right"),  # "left" is not supported for text model
+        )
+
+    @require_torch_sdpa
+    def test_sdpa_can_dispatch_on_flash(self):
+        self.skipTest(reason="CLIP text tower has two attention masks: `causal_attention_mask` and `attention_mask`")
+
+    @require_torch_sdpa
+    def test_sdpa_can_compile_dynamic(self):
+        self.skipTest(reason="CLIP model can't be compiled dynamic, error in clip_loss`")
```

Comment:
Skip the `compile dynamic` test, I found that compilation failed due to `clip_loss`, I don't know how critical this is, but I didn't find a quick fix, so just skipped for now.

Reply:
Did that happen because it outputs tuples or was it something else?

Reply:
No, loss outputs tensor. Without `compile + dynamic=True` it works fine, but I'm getting the following error in test

`_call_with_frames_removed: block: [0,0,0], thread: [31,0,0] Assertion `index out of bounds: tmp4 < ks0` failed.`

Reply:
Oh okay the error message itself doesn't seem to be super helpful. Okay with the current changes.